RTSP to HLS tutorial (#39)
membraneframework/membrane_tutorials: broadcasting/10_RTSP_Pipeline.md

> As explained in the [Architecture chapter](02_Architecture.md), the pipeline will consist of a couple of elements, that will be processing the RTP stream.

Author: done

The flow of the pipeline will consist of three steps. First, when the pipeline is initialized we will start the Connection Manager, which will set up the RTP stream via the RTSP.

> Once that is finished, we will initialize the first elements in the pipeline - the `UDP Source` and `RTP SessionBin`, which will allows us to receive RTP packets and process them.

Reviewer: ```suggestion
Once that is finished, we will set up two initial elements in the pipeline - the `UDP Source` and `RTP SessionBin`, which will allow us to receive RTP packets and process them.
```

Author: done

> When the SessionBin detects that the RTP stream has been started it will notify the pipeline with the `:new_rtp_stream` notification we will add the remaining elements to the pipeline, allowing for the whole conversion process to take place.

Reviewer: ```suggestion
When the SessionBin detects that the RTP stream has been started, it will notify the pipeline with the `:new_rtp_stream` notification. Later on, we will add the remaining elements to the pipeline, allowing for the whole conversion process to take place.
```

Author: done

> Those steps will take place respectively in the `handle_init/1`, `handle_other/3` and `handle_notification/4` callbacks.

Reviewer: ```suggestion
Those steps take place, respectively, in the: `handle_init/1`, `handle_other/3` and `handle_notification/4` callbacks.
```

Reviewer: I would also add a short sentence saying the we will go through each of these 3 callbacks and describe them in detail

Author: done
